pages/api/report/booklabels/index.tsx

```diff
@@ -111,6 +111,13 @@ const generateBarcode = async (books: Array<BookType>) => {
       };
 
       console.log("Position", pos, i);
+      
+      //Find first topic of book
+      var topics = b.topics!;
```

Comment:
die `var` würde ich mit `const` ersetzen, weil sie sich nicht ändern. In JS verwendet man `var` oder `let` extrem selten, meist tut's auch `const`

Reply:
Alles klar, wird geändert.